<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quadro</class>
 <widget class="QMainWindow" name="quadro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../imagens/img.qrc">:/img/T3Quadro.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="caixaA">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="calendario">
    <property name="geometry">
     <rect>
      <x>574</x>
      <y>142</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="volta">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>510</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(214, 102, 241, 80%);
font: 12pt &quot;Comic Sans MS&quot;;
border-radius: 8px</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../imagens/img.qrc"/>
 </resources>
 <connections/>
</ui>
